docs/fiddles/native-ui/drag-and-drop/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Drag and drop files</title>
+  </head>
+
+  <body>
+    <div class="section-wrapper">
+      <h1>Drag and drop files</h1>
+

```

Comment:
Could you add the platform and process information below the title?

Reply:
added